packages/hw-app-str/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ledgerhq/hw-app-str",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

Comment:
would you mind not changing the version? we are doing the versioning globally at the moment using lerna and our tooling will automatically update the versions

Reply:
no problem, I didn't exactly know the scheme anyway. I've changed it back.